### DISCUSSION TITLE
Run Renovate bot only for automerge

### DISCUSSION THREAD

**Discussion by glshvk** [renovatebot/renovate#19143] [Request Help] [answered]
Hi, first of all many thanks for this brilliant tool. 

We use Renovate twice a week on weekends. It works about 9-12 hours per run. 
Do we have a possibility to run Renovate for processing automerge only previously created PR's?
I want to schedule it for every hour, because the whole process takes so much time.

If so, please give me a link to the instructions

Renovate ver: 32.6.10
Platform: Bitbucket
Docker: yes

**Comment by rarkins** [comment, marked as answer]
If you define a `schedule` in Renovate config then new new branches/PRs will be created outside of it. You can also consider the `updateNotScheduled` option but that might interfere with automerging.

**Reply by glshvk** [reply]
We use external cron to run a container with Renovate.
Do you mean if I define a schedule in Renovate config on weekends and use hourly running with external cron I will get automerge process on weekdays? Am I right?

**Reply by rarkins** [reply]
Renovate's `schedule` property controls branch/PR creation schedule. `automergeSchedule` is different and defaults to "at any time". Please read the docs about `schedule` and `automergeSchedule` for further details
